lit/src/experiment-components/info/info_preview.ts

```diff
@@ -28,7 +28,11 @@ export class InfoPreview extends MobxLitElement {
     }
 
     const cleanHTML = sanitizeHtml(this.stage?.infoLines.join('\n\n'));
+    const descriptionContent = this.stage.description ? html`<div class="description">${this.stage.description}</div>` : '';
```

Comment:
nit: instead of '', use Lit's nothing

const descriptionContent = this.stage.description ? html`<div class="description">${this.stage.description}</div>` : nothing;